ofronds_internals/import.ml

```diff
@@ -0,0 +1,42 @@
+(* Standard additions to the default namespace in this project. *)
+
+include Astring
+include Bos
```

Comment:
Both `Bos` and `Astring` have fairly small namespaces, so sometimes I find this works for consistency across the project and to avoid needing many `open`s at the top of files.

Reply:
Ok, thanks! I like getting to know the motivation behind habits I've seen around.